tutorials/3d/3d_text.rst

```diff
@@ -0,0 +1,38 @@
+.. _doc_3d_text:
+
+Introduction
+============
+
+In a project there may be times when text needs to be created as
+part of a 3D scene and not just in the HUD. Godot provides two
+methods to do this. The Label3D node and the text mesh for a
+MeshInstance3D node.
+
+This page does **not** cover how to display a GUI scene in a 3D
+environment. For information on how to do that see `this <https://github.com/godotengine/godot-demo-projects/tree/master/viewport/2d_in_3d>`__
+demo project.
+
+Label3D
+-------
+
+.. image:: img/label_3d.png
+
+Label3D behaves like a label node but in a 3D space. Unlike label
+node this can not inherit properties of a GUI theme. However its
+look remains customizable and uses the same font subresource
+control nodes use.
+
+Label3D has minimal interaction with a 3D environment, it can be lit
+up and shaded by light sources if the shaded flag is enabled, but it
+will not cast a shadow, even with cast shadow turned on under the nodes
+GeometryInstance3D settings.
+
+Text mesh
+---------
+
+.. image:: img/text_mesh.png
+
+Text meshes have similarities to label3D. They display text in a 3D
+scene, and will use the same font subresource. However text is 3D and
+has the properties of a mesh. A text mesh can have a material applied
+to it, and cast shadows on the environment.
```

Comment:
Probably worth mentioning TextMesh limitations: no bitmap fonts, no fonts with self-intersections (can be result of `embolden` usage, or just non-compliant font, incorrectly designed or "optimized" by low quality tools).

And maybe also texture example for TextMesh material, to illustrate where which part of texture applied:
![texture](https://user-images.githubusercontent.com/7645683/170648952-c6ec6724-6bda-43ea-bd0f-4c3337c87cb3.png)
⇒ 
![result](https://user-images.githubusercontent.com/7645683/169757612-2285f4a4-9982-4581-8d3c-117686983c61.png)